### ISSUE TITLE
using DialogFragment

### ISSUE THREAD

**Issue by ankushsachdeva** [rockerhieu/emojicon#22]
Can we have the EmojiconsFragment class in the form of a dialog(using DialogFragment) so that we can easily replace this fragment in place of the soft keyboard. This is how whatsapp and other messaging apps smoothly toggle between qwerty keyboard and emoticons

**Comment by funcoding** [comment]
Can this feature be implemented?

**Comment by daniele-athome** [comment]
@funcoding I ended up rewriting all objects as views and used a popupwindow. You can find the code kontalk/emojicons, but please note that is not compatible with the current emojicon library.

**Comment by ankushsachdeva** [comment]
you can use ankushsachdeva/emojicon

**Comment by funcoding** [comment]
@ankushsachdeva I tried out your's and facing some issues with lollipop

**Comment by funcoding** [comment]
@daniele-athome Thanks. I shall have a look.

**Comment by dekaru** [comment]
@daniele-athome @ankushsachdeva  is it possible to get your projects from gradle?

**Comment by daniele-athome** [comment]
@dekaru I don't think so, I plan to merge my modifications to @rockerhieu repository.

**Comment by ankushsachdeva** [comment]
Not as of now, but I plan to do it in future

**Comment by bolatov** [comment]
hello @daniele-athome,

any news on merging your modifications?

**Comment by daniele-athome** [comment]
@bolatov sorry not yet. I'm trying to find the time to do it, but I can't promise anything though. I will do it :-)

**Comment by Jaymo** [comment]
@ankushsachdeva  facing issues using your library on API level >=21 any workaround ?

**Comment by aalt0** [comment]
hi, any news on this?
